packages/image_picker/example/android/app/src/main/java/io/flutter/plugins/imagepickerexample/MainActivity.java

```diff
@@ -1,18 +1,23 @@
-// Copyright 2017 The Chromium Authors. All rights reserved.
+
```

Comment:
With the new Flutter stable we no longer need this. We can just refer to `FlutterActivity` in the manifest.